portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -89,21 +89,17 @@ private String getStringParameter(HttpServletRequest request, String name, Strin
 
   /** Get number of comments to display from request */ 
   private int getNumCommentsParameter(HttpServletRequest request) {
```

Comment:
Actually, for consistency with your `getStringParameter()`, this should simply be `getIntegerProperty(request, name, defaultValue)`. That would "future-proof" some code and allow you to use this for another integer parameter if you needed (you probably don't need this for the walkthrough here, but it's a good practice).

Reply:
This still needs work